geoapps/iso_surfaces/driver.py

```diff
@@ -34,6 +34,7 @@ def run(self):
         Create iso surfaces from input values
         """
 
+        workspace = self.params.geoh5.open(mode="r+")
```

Comment:
We have a choice here to assume the workspace was opened in r+ on params before calling driver.run?